src/reducers/tests/actions/moxios_mock.jsx

```diff
@@ -0,0 +1,17 @@
+const Response = {
```

Comment:
Move this file to a different folder inside the test folder and name it fixtures

Reply:
noted